framework/interceptor.js

```diff
@@ -22,9 +22,10 @@ class Interceptor {
         return requestStopper;
     }
 
-    async abortRequestsDuringAction(action, requestUrlFragment = "") {
+    async abortRequestsAfterAction(action, requestUrlFragment = "", waitDuration = 500) {
```

Comment:
Just a note: this is breaking change, due to rename and next release will need to have major version updated. (We have merged some breaking changes already, so it will be anyways)